client/my-sites/upgrades/domain-management/dns/txt-record.jsx

```diff
@@ -57,7 +64,12 @@ const TxtRecord = React.createClass( {
 						onChange={ this.props.onChange }
 						value={ this.props.fieldValues.data }
 						placeholder={ this.translate( 'e.g. %(example)s', { args: { example: 'v=spf1 include:example.com ~all' } } ) } />
-					{ ! isDataValid ? <FormInputValidation text={ this.translate( 'Invalid TXT Record' ) } isError={ true } /> : null }
+					{ hasNonAsciiData && (
+						<div className="form-input-validation dns__warning">
```

Comment:
Heh, good point. But then I remembered that even Bootstrap (which I'd consider as sort of a "standard") has a `warning` validation and it seems like a valid use case (like literally this one 😀).
A different approach would be to use `FormSettingExplanation` - but it seems too bland/soft to use here :/
It's not a blocker and I don't want to seem pushy about it - it just seems like a missed opportunity. You basically duplicated the code of the component to implement this. I can imagine that such a warning state could be a nice addition to our components.